warehouse/integrations/manager/stats.go

```diff
@@ -0,0 +1,69 @@
+package manager
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rudderlabs/rudder-go-kit/stats"
+
+	"github.com/rudderlabs/rudder-server/utils/misc"
+	"github.com/rudderlabs/rudder-server/warehouse/internal/model"
+	warehouseutils "github.com/rudderlabs/rudder-server/warehouse/utils"
+)
+
+type statsManager struct {
```

Comment:
minor: name should be managerWithStats or statsManager?

Reply:
It's already statsManager. Do you mean should I rename to managerWithStats?